pep-8104.rst

```diff
@@ -0,0 +1,152 @@
+PEP: 8104
+Title: 2023 Term Steering Council election
+Author: Ee Durbin <ee@python.org>
+Sponsor: Brett Cannon <brett@python.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 08-Nov-2022
+
+
+Abstract
+========
+
+This document describes the schedule and other details of the December
+2022 election for the Python steering council, as specified in
+:pep:`13`. This is the steering council election for the 2023 term
+(i.e. Python 3.12).
+
+
+Election Administration
+=======================
+
+The steering council appointed the
+`Python Software Foundation <https://www.python.org/psf-landing/>`__
+Director of Infrastructure, Ee Durbin, to administer the election.
+
+
+Schedule
+========
+
+There will be a two-week nomination period, followed by a two-week
+vote.
+
+The nomination period shall be: November 14, 2022 through `November 29, 2022 12:00 UTC
+<https://www.timeanddate.com/worldclock/fixedtime.html?msg=Python+Steering+Council+nominations+close&iso=20221129T12>`_
+(the end of November 28, 2022 `Anywhere on Earth
+<https://www.ieee802.org/16/aoe.html>`_).
+
+The voting period shall be: December 1, 2022 12:00 UTC through
+`December 16, 2022 12:00 UTC
+<https://www.timeanddate.com/worldclock/fixedtime.html?msg=Python+Steering+Council+voting+closes&iso=20221216T12>`_
```

Comment:
December 1, 12:00 UTC to December 16, 12:00 UTC is 15 days.

> ... followed by a two-week
vote.

Should it be 14 days?

Reply:
I assumed that too based on the text here, but apparently, at least per the vote config below, it actually starts December 1 00:00 UTC, which would be 15.5 days.

Reply:
It seems kinda strange to use AoE for a start date as opposed to a deadline; I'm not sure I've seen it done. A start date of Dec 01 AoE would be Nov 30 10:00 UTC.